src/controller/CHIPDeviceController.h

```diff
@@ -240,6 +247,13 @@ class DLL_EXPORT DeviceController : public SessionRecoveryDelegate, public Abstr
      */
     CHIP_ERROR DisconnectDevice(NodeId nodeId);
 
+    /**
+     * @brief
+     *   Configures a new set of operational credentials to be used with this
+     *   controller.
+     */
+    CHIP_ERROR InitControllerNOCChain(const ControllerInitParams & params);
```

Comment:
Why is this being moved to a public API?  It needs to be called from `Init`, no?

Reply:
Will add a comment, it's because the `AndroidDeviceControllerWrapper` API doesn't allow yet full injection, so we have to do it after the fact...